students_projects.html

```diff
@@ -1,7 +1,17 @@
 ---
 layout: default-pages
 ---
+<style>
```

Comment:
Use class, and style it in separate css file(instead of inline styling).

Reply:
will inline styling create any problem?

Reply:
I have done internal styling here

Reply:
@sahilsaha7773 Adding inline css breaks uniformity . @fragm3  is right. Use external css.